Arrowgene.Ddon.GameServer/Characters/JobManager.cs

```diff
@@ -31,11 +31,26 @@ public JobManager(DdonGameServer server)
             _Server = server;
         }
 
-        public (IPacketStructure jobRes, IPacketStructure itemNtc, IPacketStructure jobNtc) SetJob(GameClient client, CharacterCommon common, JobId jobId, DbConnection? connectionIn = null)
+        public (IPacketStructure? jobRes, IPacketStructure? itemNtc, IPacketStructure? jobNtc) SetJob(GameClient client, CharacterCommon common, JobId jobId, DbConnection? connectionIn = null)
         {
             // TODO: Reject job change if there's no primary and secondary weapon for the new job in storage
             // (or give a lvl 1 weapon for free?)
 
+            var totalSlots = common.Equipment.GetItems(EquipType.Performance)
+                .Concat(common.Equipment.GetItems(EquipType.Visual))
+                .Where(x => x != null)
+                .ToList()
+                .Count;
+            if (totalSlots > client.Character.Storage.GetStorage(StorageType.StorageBoxNormal).EmptySlots())
```

Comment:
I guess this works, but what about items that are shared between the two job equip sets?